### PR TITLE
Ball intake subsystem

### PR DESCRIPTION
**Pull request by simon-andrews**
I'm working on getting the Talon SRX CAN library working and the `BallIntake` subsystem got lost in the clutter the first time around. Oops! :flushed:

### CODE REVIEW

File: src/org/robockets/steamworks/RobotMap.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 import edu.wpi.first.wpilibj.Encoder;
 import edu.wpi.first.wpilibj.RobotDrive;
+import edu.wpi.first.wpilibj.SpeedController;
```
Comment (UnknownSilicon) [inline src/org/robockets/steamworks/RobotMap.java]: What is this?

File: src/org/robockets/steamworks/subsystems/BallIntake.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+import org.robockets.steamworks.RobotMap;
+
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ * Rollers for taking in balls.
+ */
+public class BallIntake extends Subsystem {
+	private boolean isEnabled = false;
```
Comment (UnknownSilicon) [inline src/org/robockets/steamworks/subsystems/BallIntake.java:12]: Why are you checking if isEnabled is true or false when it never changes?

**Review by UnknownSilicon — CHANGES_REQUESTED** [review]

File: src/org/robockets/steamworks/subsystems/BallIntake.java (hunk 2 above)
Comment (simon-andrews) [reply src/org/robockets/steamworks/subsystems/BallIntake.java:12]: Doh!

I had SATs in the morning. It's been a rough day.

File: src/org/robockets/steamworks/RobotMap.java (hunk 1 above)
Comment (simon-andrews) [reply src/org/robockets/steamworks/RobotMap.java]: That shouldn't be there.

**Review by UnknownSilicon — APPROVED** [review]

**Pull request merged**